lib/routes.js

```diff
@@ -517,6 +517,10 @@ module.exports = function (app) {
     res.redirect('http://jsbin.com');
   });
 
+  app.get('/status', function(req, res) {
```

Comment:
I do feel like this belongs in a custom mount, but it's not a huge deal, and `status` is reserved (i.e. bins can't have the url). I'm going to include this.